test/new-e2e/tests/agent-platform/install-script/install_script_test.go

```diff
@@ -186,3 +209,38 @@ func (is *installScriptSuite) DogstatsdAgentTest() {
 	common.CheckInstallationInstallScript(is.T(), client)
 	is.testUninstall(client, "datadog-dogstatsd")
 }
+
+type installScriptSuiteSysVInit struct {
+	e2e.BaseSuite[environments.Host]
+}
+
+func (is *installScriptSuiteSysVInit) TestInstallAgent() {
+	containerName := "installation-target"
+	host := is.Env().RemoteHost
+	client := common.NewDockerTestClient(host, containerName)
+
+	err := client.RunContainer("464622532012.dkr.ecr.us-east-1.amazonaws.com/images/mirror/debian:bookworm-slim")
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ this feels a little weird and might look better if this was passed in as an env var?

Reply:
I'm still iterating on this bit but I feel inclined to leave this hardcoded (even if as a constant instead of inline), as it doesn't sound that different from the way the rest of the tests rely on the platforms json.

Reply:
Fair enough, would it be crazy to have it in the platforms json file?

Reply:
Not altogether unthinkable, though I'm hesitant given the fact that it would be using the same structure to represent completely different things (i.e. "os version" would have to be something like "docker", and the field used for AWS AMIs would be used for references to docker images) and that it still requires special casing in code.